src/http-api.ts

```diff
@@ -84,7 +84,7 @@ export class HttpApi {
   #setupMiddleware(): void {
     this.#api.use(cors({ exposedHeaders: 'dwn-response' }));
     this.#api.use(express.json());
-
+    this.#api.use(express.urlencoded({ extended: true }));
```

Comment:
Nit: Can you write a comment here explaining what this does and why we need it specifically? Good to know that the connect flow relies on it when someone is looking at the code in the future.

Even better would be some sort of test that would fail if this were removed, but not sure how annoying that test would be.

Reply:
I added a comment that this accepts formdata